manifests/init.pp

```diff
@@ -86,6 +86,7 @@
   $gzip_proxied                                              = 'off',
   $gzip_types                                                = undef,
   Enum['on', 'off'] $gzip_vary                               = 'off',
+  Enum['on', 'off'] $gzip_static                             = 'off',
```

Comment:
The manual suggests you can also use `always` as a valid value

Reply:
added always